stack/logging.rst

```diff
@@ -50,7 +50,7 @@ For example:
 
 In Python, two string formatting options can be used for log messages.
 The standard methods, such as :lmod:`~lsst.log.info` and :lmod:`~lsst.log.warn`, use a ``%``-format string in the message and pass in additional arguments containing variable information, which :lmod:`lsst.log` will internally merge into the message string with ``%`` formatting if the log record is to be printed.
-Another set of methods with a trailing ``f``, for example :lmod:`~lsst.log.infof` and :lmod:`~lsst.log.warnf`, can use :meth:`~str.format` string interpolation using curly braces.
+Another set of methods with a trailing ``f``, for example :lmod:`~lsst.log.infof` and :lmod:`~lsst.log.warnf`, can use `~str.format` string interpolation using curly braces.
```

Comment:
I think these should actually be functions, not modules, so removing `:lmod:` should be OK.